test/lit/passes/asyncify_enable-multivalue.wast

```diff
@@ -37,18 +37,10 @@
   ;; CHECK:      (export "asyncify_get_state" (func $asyncify_get_state))
 
   ;; CHECK:      (func $do_sleep
-  ;; CHECK-NEXT:  (local $0 i32)
-  ;; CHECK-NEXT:  (local $1 i32)
-  ;; CHECK-NEXT:  (local.set $0
-  ;; CHECK-NEXT:   (global.get $sleeping)
-  ;; CHECK-NEXT:  )
-  ;; CHECK-NEXT:  (local.set $1
+  ;; CHECK-NEXT:  (if
   ;; CHECK-NEXT:   (i32.eqz
-  ;; CHECK-NEXT:    (local.get $0)
+  ;; CHECK-NEXT:    (global.get $sleeping)
   ;; CHECK-NEXT:   )
```

Comment:
This diff looks like it is more optimized after the change. That's odd, I'd expect less.

That is, the optimizer is removing the unneeded locals (using simplify-locals + reorder-locals, which means it is running those passes here that it didn't before.

Reply:
Ah, this is a top-most runtime function, that we do not instrument. So this PR just makes us now flatten it, which is why it looks better.